src/System.Private.Xml/src/System/Xml/Core/XmlWellFormedWriter.cs

```diff
@@ -634,6 +634,15 @@ public override void WriteStartAttribute(string prefix, string localName, string
                         throw new ArgumentException(SR.Xml_EmptyLocalName);
                     }
                 }
+                // check local name
+                if (prefix == null || prefix.Length == 0)
+                {
+                    if (localName.Contains("xmlns:xs"))
+                    {
+                        localName = "xmlns";
+                        prefix = "xs";
+                    }
+                }
```

Comment:
Is this bug specific to .NET Core, or does it exist in .NET Framework as well?

Reply:
It does exist on .NET Framework as well.